src/status_im/contexts/wallet/send/select_address/view.cljs

```diff
@@ -32,7 +33,15 @@
           recipient                (rf/sub [:wallet/wallet-send-recipient])
           recipient-plain-address? (= send-address recipient)
           valid-ens-or-address?    (rf/sub [:wallet/valid-ens-or-address?])
-          contacts                 (rf/sub [:contacts/active])]
+          contacts                 (rf/sub [:contacts/active])
+          validate-address         (fn [address]
+                                     (debounce/debounce-and-dispatch
+                                      (if (and (> (count address) 0) 
+                                               (not (or (validation/ens-name? address) 
+                                                        (validation/eth-address? address))))
+                                        [:wallet/address-validation-failed address]
+                                        [:wallet/address-validation-success address])
+                                      300))]
```

Comment:
Maybe we can remove this logic from the view, and just `debounce-and-dispatch` an event with the necessary args

Reply:
It was like that before, but @vkjr suggested this